tools/pkg/engine_build_configs/bin/check.dart

```diff
@@ -0,0 +1,69 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io' as io;
+
+import 'package:engine_build_configs/engine_build_configs.dart';
+import 'package:engine_repo_tools/engine_repo_tools.dart';
+import 'package:path/path.dart' as p;
+
+// Usage:
+// $ dart bin/check.dart [/path/to/engine/src]
+
+void main(List<String> args) {
+  late final String? engineSrcPath;
+  if (args.isNotEmpty) {
+    engineSrcPath = args[0];
+  } else {
+    engineSrcPath = null;
+  }
+
+  // Find the engine repo.
+  late final Engine engine;
+  try {
+    engine = Engine.findWithin(engineSrcPath);
+  } catch (e) {
+    io.stderr.writeln(e);
+    io.exitCode = 1;
+    return;
+  }
+
+  // Find and parse the engine build configs.
+  final io.Directory buildConfigsDir = io.Directory(p.join(
+    engine.flutterDir.path, 'ci', 'builders',
+  ));
+  final List<String> errors = <String>[];
+  final BuildConfigLoader loader = BuildConfigLoader(
+    errors: errors,
```

Comment:
Just curious why errors needs to be passed in. Is that for dependency injection? If that's the case, why just errors and not configs for example?

Reply:
Oh! This is the result of an incomplete refactor. It was previously a `StringSink` that needed to be injected for tests, but as a `List<String>` it doesn't need to be injected anymore. Thanks!